### PR TITLE
Dependencies are updated

### PR DESCRIPTION
**Pull request by solomax**
@martin-g   Please do not merge, bootstrap-confirmation is broken :(((
working on it

### CODE REVIEW

**Comment by solomax** [comment]
@martin-g According to my tests this PR can be merged :))

File: bootstrap-samples/src/main/java/de/agilecoders/wicket/samples/pages/BasePage.html [hunk 1]
```diff
@@ -22,5 +22,6 @@
 
         <pre style="display:none" wicket:id="code-internal"></pre>
         <wicket:container wicket:id="footer-container"/>
+        <wicket:container wicket:id="scripts-container"/>
```
Comment (martin-g) [inline bootstrap-samples/src/main/java/de/agilecoders/wicket/samples/pages/BasePage.html]: What's the difference with `footer-container` ?
Why do we need the second ?

Comment (solomax) [reply bootstrap-samples/src/main/java/de/agilecoders/wicket/samples/pages/BasePage.html]: `domReady` scripts were not rendered for this page :(

According to my debugging:
`domReady` scripts goes to `scripts` container

So I have added
```
        add(new HeaderResponseContainer("scripts-container", "scripts"));

................

        <wicket:container wicket:id="scripts-container"/>

```

Comment (martin-g) [reply bootstrap-samples/src/main/java/de/agilecoders/wicket/samples/pages/BasePage.html]: Let's simplify it by using `"footer-container"` at https://github.com/l0rdn1kk0n/wicket-bootstrap/blob/11ad1e53a3fdf5e16195e2b794d7bfabf550ee91/bootstrap-samples/src/main/java/de/agilecoders/wicket/samples/WicketApplication.java#L162 and remove the new containers (java and html).

Comment (solomax) [reply bootstrap-samples/src/main/java/de/agilecoders/wicket/samples/pages/BasePage.html]: Done!
Thanks for the hint :)

**Comment by martin-g** [comment]
Thank you, @solomax !

**Pull request merged**